sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/RoleAssignmentsRestClient.cs

```diff
@@ -82,12 +82,8 @@ public async Task<Response<KeyVaultRoleAssignment>> DeleteAsync(string vaultBase
             switch (message.Response.Status)
             {
                 case 200:
-                    {
-                        KeyVaultRoleAssignment value = default;
-                        using var document = await JsonDocument.ParseAsync(message.Response.ContentStream, default, cancellationToken).ConfigureAwait(false);
-                        value = KeyVaultRoleAssignment.DeserializeKeyVaultRoleAssignment(document.RootElement);
-                        return Response.FromValue(value, message.Response);
-                    }
+                case 404:
```

Comment:
This is me not being familiar with .NET, does this mean that the switch cases for both `200` and `404` return the `message.Response`?

Reply:
Yeah - it's a fallthrough, same as with C/C++.

Reply:
Are we returning `204` like in a successful case or `404`? @srnagar thinks this can be confusing for the user.